app/models/host/base.rb

```diff
@@ -426,7 +426,8 @@ def check_provision_interface
     def uniq_interfaces_identifiers
       success = true
       identifiers = []
-      self.interfaces.each do |interface|
+      relevant_interfaces = self.interfaces.select { |i| !i._destroy }
```

Comment:
Nitpick, would `marked_for_destruction?` from autosave_association be better?  I realise they amount to the same thing, it's just _destroy seems like it's meant only for use with nested attrs.